plugin/src/main/kotlin/de/triplet/gradle/play/BootstrapTask.kt

```diff
@@ -59,19 +60,16 @@ open class BootstrapTask : PlayPublishTaskBase() {
     }
 
     private fun AndroidPublisher.Edits.bootstrapWhatsNew(editId: String) {
-        val versionCode = apks()
+        val maxTrack = tracks()
                 .list(variant.applicationId, editId)
-                .execute().apks
-                ?.map { it.versionCode }
-                ?.max() ?: return
-        val apkListings = apklistings()
-                .list(variant.applicationId, editId, versionCode)
-                .execute()
-                .listings ?: return
-
-        for (listing in apkListings) {
-            File(outputFolder, "${listing.language}/${ListingDetail.WHATS_NEW.fileName}")
-                    .writeText(listing.recentChanges)
+                .execute().tracks
+                ?.maxBy { TrackType.fromString(it.track) } ?: return
+
+        maxTrack.releases
```

Comment:
could we just chain the `releases` with another `?.` operator? That way we can avoid the `val maxTrack` and that elvis operator in L66 that doesn't assign anything but only returns from the method.

Reply:
I can actually make it an expression, with that change, that's fine.